grails-plugin-gsp/src/main/groovy/org/grails/plugins/web/taglib/FormTagLib.groovy

```diff
@@ -1194,4 +1196,12 @@ class FormTagLib implements ApplicationContextAware, InitializingBean, TagLibrar
 
         return requestDataValueProcessor.processUrl(request, link)
     }
+
+    @Override
+    void setConfiguration(Config co) {
+        // Some attributes can be treated as boolean, but must be converted to the
+        // expected value.
+        booleanAttributes = co.grails.tags.booleanToAttributes ?: ['disabled', 'checked', 'readonly']
```

Comment:
This is bigger written as `co.getProperty('grails.tags.booleanToAttributes', List, ['disabled', 'checked', 'readonly'])`